docs/deployment/cloud.rst

```diff
@@ -129,81 +127,79 @@ always will work is to use the ``port-forward`` command:
         $ > kuberay-head-<POD_ID>
         $ > ...
 
-Now that we have the desired pods we can expose their ports:
+Now that we have the desired pods, we can expose their ports:
 
 .. code-block::
    :caption: ports 8265 and 8888 are the the default ports for each service
 
         $  kubectl port-forward kuberay-head-<POD_ID> 8265
         $  kubectl port-forward jupyter-<POD_ID> 8888
 
-This way you will be able to access to your cluster services from localhost.
+Now you may access your cluster services from localhost.
 
-For development this is more than enough but if you are thinking in deploying in it somewhere probably you will need to
-configure the different ``ingress`` properties in the `values.yaml <https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/infrastructure/helm/quantumserverless/values.yaml>`_
-of the project with the configuration of your domain and provider. In the ``Jupyter configs`` section you have a
+For development this is more than enough, but if you are considering deploying it remotely, you will need to
+configure the various ``ingress`` properties in `values.yaml <https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/infrastructure/helm/quantumserverless/values.yaml>`_
+with the configuration of your domain and provider. In the ``Jupyter configs`` section you have a
 configuration example to expose through ``ingress`` in ``localhost`` the Jupyter service (disabled by default).
 
 * **Important**: ``nginx-ingress-controller`` is disabled by default because third party providers should provide its own Ingress controller. To use it locally you need to activate it too.
 
 .. _terraform-deployment:
 
-Terraform: deploy all the infrastructure in your favourite cloud provider
-===========================================================================
+Terraform: Deploy all the infrastructure in your preferred cloud provider
+=========================================================================
 
-This approach is very useful when you don't have anything where to deploy the infrastructure so let's go step by step.
+This approach is useful if you have a cloud cluster to which you want to deploy the infrastructure.
```

Comment:
In this case, we won't already have a cloud cluster (terraform is creating it), so maybe it's 'starting from scratch' or something along those lines?